docs/guides/migrating-react-router-app.md

```diff
@@ -0,0 +1,518 @@
+import * as React from "react";
+import { Outlet } from "@remix-run/react";
+
+export default function Root() {
+  return (
+    <html lang="en">
+      <head>
+        <meta charSet="utf-8" />
+        <link rel="icon" href="/favicon.ico" />
+        <meta
+          name="viewport"
+          content="width=device-width, initial-scale=1"
+        />
+        <meta name="theme-color" content="#000000" />
+        <meta
+          name="description"
+          content="My beautiful React app"
+        />
+        <link rel="apple-touch-icon" href="/logo192.png" />
+        <link rel="manifest" href="/manifest.json" />
+        <title>My React App</title>
+      </head>
+      <body>
+        <div id="root">
+          <Outlet />
+        </div>
+      </body>
+    </html>
+  );
+}
+```
+
+Notice a few things here:
+
+- We got rid of the `noscript` tag. We're server rendering now, which means users who disable JavaScript will still be able to see our app (and over time, as you make [a few tweaks to improve progressive enahancement](../pages/philosophy#progressive-enhancement), much of your app should still work).
+- Inside of the root element we render an `Outlet` component from `@remix-run/react`. This is the same component that you would normally use to render your matched route in a React Router app; it serves the same function here, but it's adapter for the router in Remix.
+
+<docs-warning><strong>Important:</strong> be sure to delete the `index.html` from your `public` directory after you've created your root route. Keeping the file around may cause your server to send that HTML instead of your Remix app when accessing the `/` route.</docs-warning>
+
+## Adapting your existing app code
+
+First, move the root of your existing React code into your `app` directory. So if your root app code lives in an `src` directory in the project root, it should now be in `app/src`.
+
+We also suggest renaming this directory to make it clear that this is your old code so that, eventually, you can delete it after migrating all of its contents. The beauty of this approach is that you don't have to do it all at once for your app to run as usual. In our demo project we name this directory `old-app`.
+
+Lastly, in your root `App` component (the one that would have been mounted to the `root` element), remove the `<BrowserRouter>` from React Router. Remix takes care of this for you without needing to render the provider directly.
+
+## Creating a catch-all route
+
+Remix needs routes beyond the root route to know what to render in `<Outlet />`. Fortunately you already render `<Route>` components in your app, and Remix can use those as you migrate to use our [routing conventions](./routing.md).
+
+To start, create a new directory in `app` called `routes`. In that directory, create a file called `$.jsx`. This is called a **catch-all route** and it will be useful to let your old app handle routes that you haven't moved into the `routes` directory yet.
```

Comment:
```suggestion
To start, create a new directory in `app` called `routes`. In that directory, create a file called `$.jsx`. This is called [a **catch-all route**](https://remix.run/docs/en/v1/guides/routing#splats) and it will be useful to let your old app handle routes that you haven't moved into the `routes` directory yet.
```

Reply:
This needs to be a relative path. Will update in a separate commit 👍